.github/CODEOWNERS

```diff
@@ -168,7 +167,6 @@
 # Kapu1178
 
 /code/modules/surgery/bodyparts/ @Kapu1178
-/code/modules/surgery/organs/ @Kapu1178
```

Comment:
what's with the removals

Reply:
they were moved to the multiple owners section